README.md

```diff
@@ -7,20 +7,15 @@
 
 This is a [Kodi](https://kodi.tv) input stream add-on wich acts as a demuxer for segmented, multi-bitrate internet streams. The most common streaming protocols such as MPEG-DASH, HLS and Microsoft Smooth Streaming are supported.
 
-The add-on also has support for DRM protected streams, such as Google Widevine, Microsoft PlayReady and others, for some of them are only available on specific operating systems.
+The add-on also has support for DRM protected streams, such as Google Widevine, Microsoft PlayReady and others, however some are only available on specific operating systems. To use the Google Widevine DRM on non-Android systems is required the installation of the Widevine CDM module library not included with this add-on.
 
-The add-on can be used from all video add-ons by appropriately configuring the *ListItem* properties to instruct Kodi to play the content by using this add-on.
-For test purposes is possible create the STRM files, where you can add the URL for the test stream to be played and the parameters to instruct Kodi to play the content by using this add-on.
+To enable your video add-on to instruct Kodi to playback by using this InputStream add-on, you have to appropriately configuring the *ListItem* properties.
 
-*Please refer to the [Wiki pages](https://github.com/xbmc/inputstream.adaptive/wiki), to read detailed instructions in how to integrate it in your add-on, how to make tests, how to build it and many other information.*
+For testing purposes it is possible to create the STRM files that contains the URL for the test stream as well the playback parameters to instruct Kodi to play the content by using this InputStream add-on.
 
-## Features in WIP
+*Please refer to the [Wiki pages](https://github.com/xbmc/inputstream.adaptive/wiki), to read detailed instructions in how to integrate it in your add-on, how to make tests, how to build it and many other information.*
```

Comment:
Please refer to the [Wiki pages](https://github.com/xbmc/inputstream.adaptive/wiki) for detailed instructions on how to integrate, build and test with your add-on.